openedx/tests/xblock_integration/xblock_testcase.py

```diff
@@ -391,10 +473,21 @@ def render_block(self, block_urlname):
         '''
         section = self._containing_section(block_urlname)
         html_response = collections.namedtuple('HtmlResponse',
-                                               ['status_code'])
+                                               ['status_code',
+                                                'content',
+                                                'debug'])
         url = self.scenario_urls[section]
         response = self.client.get(url)
+
         html_response.status_code = response.status_code
+        response_content = response.content.decode('utf-8')
+        html_response.content = self.extract_block(response_content,
```

Comment:
Should the TODO above be removed?

To do: Implement returning the XBlock's HTML. This is an XML
         selector on the returned response for that div.